cmd/bom/README.md

```diff
@@ -0,0 +1,76 @@
+# bom (Bill of Materials)
```

Comment:
For consistency, we should clarify how we want to spell the abbreviation, like:
- `bom` when referring to the binary
- BOM when referring to "Bill of Materials"

Reply:
I've seen that as of late SBOM has been gaining more traction and BOM is getting left behind. I used SBOM to abbreviate a generic bill of materials. I will set all instances of `bom` in backticks to refer to the binary